### ISSUE TITLE
how to deploy computer vision model in edgex foundry

### ISSUE THREAD

**Issue by gaurav8222** [edgexfoundry/edgex-ui-go#532]
Hi,
     I want to deploy my computer vision tensorflow model as an API  in edgex foundry and want to trigger that api on the basis of core data .

Please can you provide any pdf or any tutorial use case through which i can configure these things. It will be really helpful for me .

**Comment by badboy-huaqiao** [comment]
@gaurav8222 yeah, we have made a sample and guide of AI+EdgeX, but it's only written by Chinese, so where are you from? can you read Chinese guide?

**Comment by gaurav8222** [comment]
I want to deploy the tensorflow model also in edgex foundry app service and then using the schduler i want to triggee that app service
